pages/about/policies.html

```diff
@@ -11,6 +11,15 @@
     <script src="../../semantic/out/semantic.min.js"></script>
     <link rel="icon" href="../../assets/favicon.ico" type="assets/image/x-icon" />
 
+    <script>  function showModal(whichModal){
+      $('.ui.modal.' + whichModal).modal('show');
+      }
+    </script>
+       <script type="text/javascript">
+           function includeHTML(){
```

Comment:
can we move this bit here to the script above?

Reply:
yep, done :)